buckifier/targets_cfg.py

```diff
@@ -41,6 +41,6 @@
 """
 
 fancy_bench_template = """
-fancy_bench_wrapper(suite_name="{name}", binary_to_bench_to_metric_list_map={bench_config}, slow={slow}, expected_runtime={expected_runtime})
+fancy_bench_wrapper(suite_name="{name}", binary_to_bench_to_metric_list_map={bench_config}, slow={slow}, expected_runtime={expected_runtime}, sl_iterations={sl_iterations}, regression_threshold={regression_threshold})
```

Comment:
Should we set a default value here?
Is there any other parameter we could use?
Is the setting per test or test_suit?

Reply:
The json's contain default parameters, these are per test suite.

Reply:
would be great to have per test config. (maybe with later diff).